cicd/metrics_test.csv

```diff
@@ -0,0 +1,6 @@
+testmetric0,now-90d,now,"131.45590577322224,127.06863068860355,159.53343696489338",approx
```

Comment:
is it possible to add a comment in this file at the top and explain what these csv separated values mean ?